fix-recurring-pay

IscoRuta98 opened the pull request:
This PR aims to fix the following tickets:
- [NW-217: UI: Recurring Payment - Set Schedule - Dropdown](https://ntls.atlassian.net/jira/software/projects/NW/boards/1?selectedIssue=NW-217)
- [NW-218: Back Button on Recurring Payments Screen (After Creating Recurring Payment)](https://ntls.atlassian.net/jira/software/projects/NW/boards/1?selectedIssue=NW-218)

billguo99 commented:
Getting the following error when trying to add a recurring payment:
![image](https://github.com/ntls-io/nautilus-wallet/assets/31125065/f698f95a-dc99-46b4-917f-707e25b93317)
I think the frequency input is not a valid integer, think we might be giving it a string.

billguo99 commented:
![image](https://github.com/ntls-io/nautilus-wallet/assets/31125065/ddddb8f2-429a-443c-ba95-3054a2c7e2dd)
Actually it's not getting passed to the request.

IscoRuta98 commented:
> Getting the following error when trying to add a recurring payment: ![image](https://user-images.githubusercontent.com/31125065/254289358-f698f95a-dc99-46b4-917f-707e25b93317.png) I think the frequency input is not a valid integer, think we might be giving it a string.

I fixed this error, I accidently removed the formControlName. I've added it back.
